### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by MishaLehotskyi**
[DEMO LINK](https://MishaLehotskyi.github.io/react_list-of-goods-hooks/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,7 +1,36 @@
...
+  reverse: boolean,
+};
+
+function getGoods(goods: string[], sortParams: SortParams) {
+  const goodsCopy = [...goods];
+
+  if (sortParams.property) {
+    goodsCopy.sort((good1, good2) => {
+      switch (sortParams.property) {
+        case 'name':
```
Comment (VitaliyBodnar) [inline src/App.tsx:17]: please move 'name' and 'length' to the `SortType` enum as it will make a code cleaner

File: src/App.tsx [hunk 2]
```diff
@@ -16,46 +45,76 @@ export const goodsFromServer = [
...
           type="button"
-          className="button is-info is-light"
+          className={cn(
+            'button is-info',
+            { 'is-light': sortParams.property !== 'name' },
+          )}
+          onClick={() => setSortParams({
+            ...sortParams,
+            property: 'name',
+          })}
```
Comment (VitaliyBodnar) [inline src/App.tsx:66]: it is bad practise to make anonymous arrow functions in jsx.
make handler in component and pass reference here

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (anastasiiavorobiova) [reply src/App.tsx:23]: Not fixed from the previous review

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done!

**Review by anastasiiavorobiova — APPROVED** [review]
Awesome work!

**Pull request closed without merging**